tests/tests.py

```diff
@@ -760,5 +760,108 @@ def test_template_free_mode(self):
         self.check_examples()
 
 
+class TestRnaview(R2dtTestCase):
+    """Check that the RNAVIEW script works correctly."""
+    pdb_file = Path("examples") / "PZ1_Bujnicki_1.pdb"
+    test_results = Path("tests") / "results" / "rnaview"
+    precomputed_results = Path("tests") / "examples" / "rnaview"
+    files = ["PZ1_Bujnicki_1.colored.svg"]
+    output_file = Path("examples") / "PZ1_Bujnicki_1.colored.svg"
+    cmd = f"python ./utils/rnaview.py {pdb_file} "
+
+    def setUp(self):
+        if not self.test_results.exists():
+            print(f"Creating test results directory: {self.test_results}")
+            self.test_results.mkdir(parents=True)
+        print(f"Running command: {self.cmd}")
+        result = runner.run(self.cmd, print_output=True)
+        print("RNAVIEW output:", result)
+
+        if self.output_file.exists():
+            print(f"Output file generated: {self.output_file}")
+            print(f"Copying output file to: {self.test_results}")
+            shutil.copy(self.output_file, self.test_results)
+        else:
+            print(f"Output file not found: {self.output_file}")
+
+
+    def test_rnaview_output_exists(self):
+        """Check that the RNAVIEW script generates the expected output file."""
+        print(f"Checking existence of output file in: {self.test_results}")
+        self.check_examples()
+
+    def test_rnaview_output_matches(self):
+        """Check that the generated file matches the precomputed file."""
+        print(f"Checking generated file against precomputed file in: {self.precomputed_results}")
+        self.check_examples()
+
+
+class TestAnimateBulk(R2dtTestCase):
```

Comment:
Would it be possible to add a test case for non-bulk animation as well?

Similar to this command from the docs:

```
python3 utils/animate3d.py \
    examples/PZ1_solution_0.pdb \
    examples/PZ1_Bujnicki_1.pdb \
    examples/PZ1_solution_0_to_PZ1_Bujnicki_1.animated.svg
```